sql/core/src/test/resources/sql-tests/results/window.sql.out

```diff
@@ -1054,3 +1054,219 @@ b	5	NULL	x	y	z	x	z
 a	6	z	x	y	z	x	v
 a	7	v	x	y	z	x	v
 a	8	NULL	x	y	z	x	v
+
+
+-- !query
+SELECT DENSE_RANK() OVER (
+  ORDER BY salary
+ )
+FROM basic_pays
+-- !query schema
+struct<DENSE_RANK() OVER (ORDER BY salary ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW):int>
+-- !query output
+1
+10
+11
+12
+13
+14
+15
+16
+17
+2
+3
+4
+5
+6
+7
+8
+9
+
+
+-- !query
+SELECT DENSE_RANK() OVER (
+  ORDER BY salary ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW
+ )
+FROM basic_pays
+-- !query schema
+struct<DENSE_RANK() OVER (ORDER BY salary ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW):int>
+-- !query output
+1
+10
+11
+12
+13
+14
+15
+16
+17
+2
+3
+4
+5
+6
+7
+8
+9
+
+
+-- !query
+SELECT DENSE_RANK() OVER (
+  ORDER BY salary ROWS BETWEEN 5 PRECEDING AND CURRENT ROW
+ )
+FROM basic_pays
+-- !query schema
+struct<DENSE_RANK() OVER (ORDER BY salary ASC NULLS FIRST ROWS BETWEEN 5 PRECEDING AND CURRENT ROW):int>
```

Comment:
It seems to be that we return the wrong answer here. `5 PRECEDING` is not respected.

Reply:
Thus, I'm -1 to this change.

Reply:
All right, PG's behavior look not reasonable. Close this PR